src/Models/CategoryModel.php

```diff
@@ -16,5 +15,14 @@ public static function getCategories(PDO $db): array
         $query->execute();
         return $query->fetchAll();
     }
+    public static function getCategoryTitle(PDO $db, int $id): ProductsEntity
+    {
+        $sql = 'SELECT `categories`.`name` FROM `categories` JOIN `products`
+        ON `products`.`category_id` = `categories`.`id` WHERE `category_id` = :id;';
+        $query = $db->prepare($sql);
+        $query->setFetchMode(PDO::FETCH_CLASS, ProductsEntity::class);
+        $query->execute(["id"=>$id]);
+        return $query->fetch();
+    }
```

Comment:
Why do you return a `ProductsEntity` when what the function says it does is to get a Category Title?

Your SQL query is a lot more complicated than it needs to be.